modules/machineset-enabling-efa-options.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-aws.adoc
+
+:_content-type: PROCEDURE
+[id="machineset-enabling-efa-options_{context}"]
+= Enabling MPI workloads that use an AWS Elastic Fabric Adapter
+
+After configuring a machine set to support the use of an AWS Elastic Fabric Adapter (EFA), you must install additional software to run MPI workloads. 
+
+For more information about using Kubeflow and the MPI Operator in {product-title} and an example, see link:https://cloud.redhat.com/blog/how-to-use-kubeflow-and-the-mpi-operator-on-openshift[How to use Kubeflow and the MPI Operator on OpenShift].
+
+.Prerequisites
+
+* You have configured a machine set to support the use of an EFA.
+
+.Procedure
+//As far as I know, these don't actually have to be in any particular order. Is there an order that makes sense?
```

Comment:
@JoelSpeed I think this is my only outstanding Q for you. Thoughts?

Reply:
As far as I know, no ordering needed, current order looks ok to me